src/wrapper/clap-juce-wrapper.cpp

```diff
@@ -1014,6 +1014,7 @@ class ClapJuceWrapper : public clap::helpers::Plugin<clap::helpers::Misbehaviour
 #if JUCE_LINUX
     bool guiX11Attach(const char *displayName, unsigned long window) noexcept
     {
+        (void)displayName;
```

Comment:
i guess this is for an "unused argument" warning? We've been preferring `juce::ignoreUnused(...)` for that purpose, just to stay consistent with the JUCE codebase.

Reply:
Yes, every project I work on handles it differently, so often mix them.